asciidoc-markup-sample-doc/asciidoc-markup-samples.adoc

```diff
@@ -4,6 +4,11 @@
 
 = AsciiDoc Mark-up Quick Reference for Red{nbsp}Hat Documentation
 
+[NOTE]
+====
+Some markup examples used in this overview are based on new Asciidoctor features and they are not a part of the standard set of elements. Please, use the `:experimental:` tag in the header of your document to enable this functionality.
```

Comment:
Do you know whether ccutil/Pantheon has this attrib. set by default? If so, it might be worth mentioning.

Reply:
I do not. But it can be tested, I suppose.